block_my_feedback.php

```diff
@@ -57,14 +57,7 @@ public function get_content() : stdClass {
         $this->content->footer = '';
 
         $template = new stdClass();
-        $template->allfeedbackurl = new moodle_url('/report/myfeedback/index.php');
         $template->feedback = $this->fetch_feedback();
-
-        // Hide the block when no content.
-        if (!$template->feedback) {
-            return $this->content;
-        }
```

Comment:
I thought that was a feature. how come we're removing?

Reply:
good to flag, hope reasons made sense.

Reply:
re-add this, mark as ready for review & I'll merge